### DISCUSSION TITLE
codecov links in the issues with missing tests

### DISCUSSION THREAD

**Discussion by 00thirdeye00** [cieslarmichal/faker-cxx#797] [General]
Hej, I tried couple of links now in the issues missing test cases and found it doesn't work. Is there a way to update links or place where we can see codecov by ourselves for that specific issue?

**Comment by cieslarmichal** [comment]
here is a main branch codecov: https://app.codecov.io/github/cieslarmichal/faker-cxx

**Reply by 00thirdeye00** [reply]
does this link update after each PR or this is permalink for each state?

**Reply by cieslarmichal** [reply]
Link stays as the same url but the coverage there is updated on each changes

**Reply by 00thirdeye00** [reply]
so its updated after the merge I guess

**Reply by cieslarmichal** [reply]
Yes

**Reply by 00thirdeye00** [reply]
perfect, I see it now.
